modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -15,16 +15,16 @@ Anypoint Service Mesh is developed using Istio and Kubernetes. Therefore, you mu
 [[supported_versions]]
 == Software Requirements
 
-Your Anypoint Service Mesh installation requires the following applications and Kubernetes environments:
+Your Anypoint Service Mesh installation requires you to:
 
-* One of the following environments on which to run Kubernetes:
+* Have installed one of the following environments on which to run Kubernetes:
 ** Google Kubernetes Engine (GKE)
 ** Amazon EKS
 ** Azure Kubernetes Service (AKS)
 ** Red Hat OpenShift
-* Kubernetes (versions 1.12 through 1.20) or Red Hat OpenShift (version 4.x)
-
-* <<install_Istio,Istio>> (currently supported versions include 1.7.x, 1.8.x, 1.9.x, and 1.10.x)
+* Have installed Kubernetes (versions 1.12 through 1.20) or Red Hat OpenShift (version 4.x)
+* Have installedd <<install_Istio,Istio>> (currently supported versions include 1.7.x, 1.8.x, 1.9.x, and 1.10.x)
+* Not have installed https://svc-cat.io/[Service Catalog^] in the target cluster
```

Comment:
Prob say separately "Do not install" 

And prob should tell them what to do if they have already installed the Service Catalog. Uninstall? Anything else?

Reply:
Done! Thanks!